importing-reports/crystal-reports-converter.md

```diff
@@ -87,6 +87,14 @@ The output from the *gacutil.exe* above shows that the current version of the in
           <assemblyIdentity name="CrystalDecisions.ReportAppServer.ReportDefModel" publicKeyToken="692fbea5521e1304" culture="neutral"/>
```

Comment:
I think lines CrystalDecisions.ReportAppServer.ReportDefModel and 95 are duplicate